File: ClayInspectionScheduler/Scripts/app/QuickRemark.ts
namespace InspSched
{
  interface IQuickRemark
  {
    Remark: string;
    Commercial: boolean;
    Building: boolean;
    Electrical: boolean;
    Mechanical: boolean;
    Plumbing: boolean;
    PrivateProvider: boolean;
  }
  export class QuickRemark implements IQuickRemark
  {
    public Remark: string;
    public Commercial: boolean;
    public Building: boolean;
    public Electrical: boolean;
    public Mechanical: boolean;
    public Plumbing: boolean;
    public PrivateProvider: boolean;

    constructor()
    {

    }


  }
}

File: ClayInspectionScheduler/Models/QuickRemark.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ClayInspectionScheduler.Models
{
  public class QuickRemark
  {
    public string Remark { get; set; }
    public bool Commercial { get; set; }
    public bool Building { get; set; }
    public bool Electrical { get; set; }
    public bool Mechanical { get; set; }
    public bool Plumbing { get; set; }
    public bool PrivateProvider { get; set; }
    public QuickRemark()
    {

    }

    public static List<QuickRemark> GetInspectionQuickRemarks()
    {
      string query = @"
        SELECT 
          quick_remark Remark,
          commercial Commerical,
          building Building,
          electrical Electrical,
          mechanical Mechanical,
          plumbing Plumbing,
          private_provider PrivateProvider
        FROM bpInspectionQuickRemarks";
      return Constants.Get_Data<QuickRemark>(query);
    }

    public static List<QuickRemark> GetCachedInspectionQuickRemarks()
    {
      return (List<QuickRemark>)MyCache.GetItem("quickremarks");
    }
  }
}